pset-iterators.js

```diff
@@ -66,6 +72,12 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.find(funsction (president){
+  return president.president.split("")[0] === "James"
+})
+
+console.log(presidentsNamedJames)
+
```

Comment:
@ChicoMangusta  Good start to this assignment here. I have a few suggestions for you
- For this example, instead of using `find` that will return only the first president with the name `James` you can use the `filter` method. 
- Also for the president split currently when you don't include a space in your split method, ex. `president.president.split("")[0]` it will split every single letter in the string. you would want to add a space `president.president.split(" ")[0]`.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)